tests/Resources/Page/ODH/AiApps/Anaconda.resource

```diff
@@ -107,81 +107,36 @@ Anaconda Activation Should Have Failed
-  ${version-check}=  Is RHODS Version Greater Or Equal Than  1.11.0
-  IF  ${version-check}==False
-    Page Should Not Contain Element     xpath://div[@class="pf-c-card__title"]/span[.="Anaconda Commercial Edition"]
-  ELSE
-    Page Should Not Contain Element     xpath://div[@class="pf-c-card__title"]/span[.="Anaconda Professional"]
-  END
+  Page Should Not Contain Element     xpath://div[@class="pf-c-card__title"]/span[.="Anaconda Professional"]
 
 Verify Service Is Available In The Explore Page Based On Version
   [Documentation]   Checks service is available in explore page based on version.
-  ${version-check}=  Is RHODS Version Greater Or Equal Than  1.11.0
-  IF  ${version-check}==False
-    Verify Service Is Available In The Explore Page    Anaconda Commercial Edition
-  ELSE
-    Verify Service Is Available In The Explore Page    Anaconda Professional
-  END
+  Verify Service Is Available In The Explore Page    Anaconda Professional
 
 Verify Service Provides "Get Started" Button In The Explore Page Based On Version
   [Documentation]    Checks service provides "get started" button in explore based on version.
-  ${version-check}=  Is RHODS Version Greater Or Equal Than  1.11.0
-  IF  ${version-check}==False
-    Verify Service Provides "Get Started" Button In The Explore Page    Anaconda Commercial Edition
-  ELSE
-    Verify Service Provides "Get Started" Button In The Explore Page    Anaconda Professional
-  END
+  Verify Service Provides "Get Started" Button In The Explore Page    Anaconda Professional
 
 Verify Service Provides "Enable" Button In The Explore Page Based On Version
   [Documentation]   Checks service provides "enable" button in explore page based on version
-  ${version-check}=  Is RHODS Version Greater Or Equal Than  1.11.0
-  IF  ${version-check}==False
-    Verify Service Provides "Enable" Button In The Explore Page     Anaconda Commercial Edition
-  ELSE
-    Verify Service Provides "Enable" Button In The Explore Page     Anaconda Professional
-  END
+  Verify Service Provides "Enable" Button In The Explore Page     Anaconda Professional
 
 Verify Anaconda Service Is Enabled Based On Version
   [Documentation]   Checks service is enabled based on version.
-  ${version-check}=  Is RHODS Version Greater Or Equal Than  1.11.0
-  IF  ${version-check}==False
-    Verify Service Is Enabled   Anaconda Commercial Edition
-  ELSE
-    Verify Service Is Enabled   Anaconda Professional
-  END
+  Verify Service Is Enabled   Anaconda Professional
 
 Verify Page Contain Anaconda Based On Version
   [Documentation]    Checks page contain Anaconda text or not based on version.
-  ${version-check}=  Is RHODS Version Greater Or Equal Than  1.11.0
-  IF  ${version-check}==False
-    Wait Until Page Contains    Anaconda Commercial Edition     timeout=30
-  ELSE
-    Wait Until Page Contains    Anaconda Professional           timeout=30
-  END
+  Wait Until Page Contains    Anaconda Professional           timeout=30
```

Comment:
since the version check is not necessary anymore, you can delete this kw and use `Verify Service Is Available In The Explore Page` kw instead, directly in the test cases